app/views/partials/multi_factor_authentication/_mfa_selection.html.erb

```diff
@@ -2,6 +2,8 @@
       'two_factor_options_form[selection][]',
       option.type,
       false,
+      # disabled: option.disabled? && option.type === 'piv_cac' || option.disabled? && option.type === 'backup_code' || option.disabled? && option.type === 'phone',
+      # checked: option.disabled? && option.type === 'piv_cac' || option.disabled? && option.type === 'backup_code',
```

Comment:
Is this still in progress? I wouldn't expect to see commented code here.

Reply:
That was left there by mistake. It is removed.